### PR TITLE
Add configurable assets by env

### PR DESCRIPTION
**Pull request by 3zk1m0**
# What Does This PR Do
Implement configurable assets by env.
Using env flag VITE_ASSET_SET, you can choose whether to load assets "x" or "y". You set these at assetSetStore, right now there are "fdf" and "neutral" assets.

When you want to use dynamic assets you want to make sure your dynamic asset has equivalent for sets (both "fdf" and "neutral", so on). If the assets are images, you want to then import them to your components from /assets/set/* where the env-defined assets are placed by VITE on startup. If the assets are translation keys, make sure keys are present in locales under assetSetStore, and then use the "dynamic" namespace for accessing them.

# Why
- [The feature request](https://github.com/pvarki/rasenmaeher-ui/issues/20)
TLDR make sure we can have the "FDF" facade with FDF-copyrighted stuff and "neutral" + potentially customisable stuff for other facades.

# Images of changes
### Neutral facade, login screen
Näyttökuva 2024-7-6 kello 14 35 07

FDF facade has the FDF logos for top bar and center, and FDF copyright notices & privacy policy.

### CODE REVIEW

**Comment by benjamsf** [comment]
Alright, I think I have all set here

**Comment by benjamsf** [comment]
@3zk1m0 as the PR is opened by you, the review request is borked but here, review ready!

**Comment by benjamsf** [comment]
Fixed few mishaps and also this small content bug: https://github.com/pvarki/rasenmaeher-ui/issues/30

Now ready for review & merge if all good, @3zk1m0 when you can

**Review by benjamsf — APPROVED** [review]
@3zk1m0 checked this & told in chat that this one's OK, so merging.
